tests/integration/keras/test_lr.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+import pytest
+import tensorflow as tf
+from finetuner.tuner.keras import KerasTuner
+
+
+@pytest.mark.parametrize('scheduler_step', ['batch'])
```

Comment:
guess you want to add `epoch` to here

Reply:
I don't. Keras scheduler is stepped automatically - I mention this in docstrings